_pages/oidc/getting-started.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: OpenID Connect
 lead: >
-   [OpenID Connect](http://openid.net){:class="usa-link--external"} is a simple identity layer built on top of the OAuth 2.0 protocol. Login.gov supports [version 1.0](http://openid.net/specs/openid-connect-core-1_0.html){:class="usa-link--external"} of the specification and conforms to the [iGov Profile](https://openid.net/wg/igov){:class="usa-link--external"}.
+   [OpenID Connect](http://openid.net){:class="usa-link--external"} (OIDC) is a simple identity layer built on top of the OAuth 2.0 protocol. Login.gov supports [version 1.0](http://openid.net/specs/openid-connect-core-1_0.html){:class="usa-link--external"} of the specification and conforms to the [iGov Profile](https://openid.net/wg/igov){:class="usa-link--external"}.
```

Comment:
If you want this update to show up in all of the OIDC pages in the sidebar, you will need to update the lead in those pages (such as oidc/certifiactes and oidc/authorization) as well.

Reply:
Never mind, I think you got them all except for /certificates I think might be missing

Reply:
Thank you!